en/SettingsView/MAVLink.md

```diff
@@ -1,9 +1,63 @@
 # MAVLink Settings
 
 The MAVLink settings (**SettingsView > MAVLink**) allow you to configure options and view information specific to MAVLink communications.
+This includes setting the MAVLink system ID for *QGroundControl*, viewing link quality, and managing log uploads (e.g. to [Flight Review](https://logs.px4.io)).
 
-## Link Status
+![MAVLink settings scren](../../assets/settings/mavlink/overview.jpg)
 
-This shows you the status of MAVLink message transfer over the communications link. A high **Loss rate** could lead to protocol errors with things like parameter download or mission upload/download.
+## Ground Station {#ground_station}
 
-![Link Status](../../assets/settings/MAVLinkLinkStatus.jpg)
+This section sets the MAVLink properties and behaviour of *QGroundControl*.
+
+![Ground Station](../../assets/settings/mavlink/ground_station.jpg)
+
+- **MAVLink System ID:** System ID of *QGroundControl* on the MAVLink network (Default: 255).
+  Vehicles are typically allocated IDs from 1. 
+  You may have to specify another ID if there are multiple ground stations or MAVLink applications on the network. 
+- **Emit heartbeat:** Disable emission of regular MAVLink HEARTBEAT message (Default: True).
+  MAVLink systems are required to emit heartbeats by the protocol, but this is not strictly speaking *needed* on single-vehicle/GCS networks.
+- **Request start of MAVLink telemetry streams (ArduPilot only):** TBD (Default: True).
+- **Only accept MAVs with the same protocol version:** TBD (Default: True).
```

Comment:
Is this MAVLInk1 vs MAVLink 2 or the specific version as in the xml file (ie 2.3 currently)?
What is implication of this being true vs false in real life?

Reply:
Mavlink 1 or 2.